wp-content/themes/borderzine/partials/widget-content.php

```diff
@@ -43,6 +43,12 @@
 				<?php echo get_the_post_thumbnail( get_the_ID(), 'large', $img_attr); ?>
 			</a>
 		<?php
+	} elseif( $thumb == 'rect_thumb_half' ){
+		$img_attr = array();
+		$img_attr = 'attachment-rect-thumb-half';
```

Comment:
Variable first set to array, then to string?

Reply:
🤦‍♂ The `['class']` portion of that is missing. Updated in https://github.com/INN/umbrella-borderzine/pull/79/commits/631b52fcf32d2f85d4594ff246be2859cf69f6c5